docs/cues/cue-playback.md

```diff
@@ -226,34 +248,20 @@ the same fader group from another page
 \[Release All Playbacks In This Group\] - releases any active playback in
 the fader group.
 
+The release will use the [Temporary Release Mask](#temporary-release-mask) as described above. 
+
 > **"Fader Group"** refers to a particular area of the console, for example
 the main faders below the screen on a Tiger Touch or the top row on the
 left hand side of the Arena.
 
-Release Mask
-------------
-
-You can release individual fixtures or attributes, and remove fixtures
-from shapes in playbacks using the release mask. There is a Global
-release mask which affects all release operations, or you can set a
-Local release mask which affects only the current playback.
+### Releasing the programmer contents
```

Comment:
Might be worth mentioning the `<Release><@A>` syntax? I had no idea it existed until I looked it up!

Reply:
Sorry can you explain, I do not know what that does

Reply:
Worked it out

Reply:
So if I have a fixture selected that has C, M and Y in the programmer, and I am on the `Colour` attribute group, I can do `<Release><@A>` and the Cyan information for the selected fixture will be released from the programmer. This can be useful if you do not have clear release enabled but you do want to release a given attribute value.